va/validation-authority.go

```diff
@@ -208,6 +237,15 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	z := sha256.Sum256(RS)
 	zName := fmt.Sprintf("%064x.acme.invalid", z)
 
+	// Check for resolver SERVFAIL for A/AAAA records
```

Comment:
I feel like these two checks are what's keeping the rest of this valuable PR from landing.

Would you be opposed to pulling these and we handle it when we rework the logging of where we connect resolution changes?